firefox-ios/Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -113,9 +113,9 @@ class ClearPrivateDataTableViewController: ThemedTableViewController {
             cell.selectionStyle = .none
             control.tag = indexPath.item
         } else {
-            cell.textLabel?.text = .SettingsClearPrivateDataClearButton
-            cell.textLabel?.textAlignment = .center
-            cell.textLabel?.textColor = currentTheme().colors.textWarning
+            cell.centeredButtonLabel.text = .SettingsClearPrivateDataClearButton
+            cell.centeredButtonLabel.textAlignment = .center
```

Comment:
Do we still need to set the text alignment to center here - especially with it being a centered button label?